src/main/java/com/microsoft/sqlserver/jdbc/SQLServerPreparedStatement.java

```diff
@@ -2681,13 +2681,6 @@ private void checkValidColumns(TypeInfo ti) throws SQLServerException {
 
     private void checkAdditionalQuery() {
         while (checkAndRemoveCommentsAndSpace(true)) {}
```

Comment:
At this point the method is just:

```
private void checkAdditionalQuery() {
        while (checkAndRemoveCommentsAndSpace(true)) {}
    }
```

May as well replace the two `checkAdditionalQuery()` method calls at 2317 and 2512, with `while (checkAndRemoveCommentsAndSpace(true)) {};`

Reply:
No, I'll be duping the code.